src/css/properties/background.zig

```diff
@@ -524,63 +524,54 @@ pub const AspectRatio = struct {
 };
 
 pub const BackgroundProperty = packed struct(u16) {
-    @"background-color": bool = false,
```

Comment:
it's clearer to keep it as the exact CSS property name. Instead of having to mentally ask "is it background-color or is it color?

Reply:
i presumed that was clear since it was on `BackgroundProperty` and i needed the fields to not interfere with the decl shorthands which are used heavily